### DISCUSSION TITLE
Selector is not pure from a third party package using app-router

### DISCUSSION THREAD

**Discussion by emanuelcoelho1986** [vercel/next.js#62928] [Help]
### Summary

Hello!

I hope everyone is well.

I'm trying to migrate a web app from Next.js 12 to 13 to use the new `App Router`. For context, currently, the app has no routes. Everything is a state to show and hide components (-- ugly IMO --).

When I look at the migration guide, as soon as I create the `app` folder (even if empty), I start getting `./node_modules/@example/package-web/lib/components/Inputs/DateField/style.module.css:13:1
Syntax error: Selector "input::-webkit-outer-spin-button,
input::-webkit-inner-spin-button" is not pure (pure selectors must contain at least one local class or id)` from a third-party package.

I can't find information about the issue in the documentation. I can only find info on how to fix this if it is our own CSS module.

Do I need to jump into a custom css-loader? Is there a config in Webpack I can use to fix this (that I missed from the docs)?

### Additional information

I added the package to the `transpilePackages`, but it didn't work. 😞 

Here is the error message

```js
./node_modules/@example/package-web/lib/components/Inputs/DateField/style.module.css:13:1
Syntax error: Selector "input::-webkit-outer-spin-button,
input::-webkit-inner-spin-button" is not pure (pure selectors must contain at least one local class or id)

  11 | }
  12 | 
> 13 | input::-webkit-outer-spin-button,
     | ^
  14 | input::-webkit-inner-spin-button {
  15 |   -webkit-appearance: none;
```
```

**Comment by icyJoseph** [comment]
Do you have a custom PostCSS configuration? I wonder if its not touching the app folder?

**Reply by emanuelcoelho1986** [reply]
I have:

```
const { join } = require('path');

module.exports = {
  plugins: {
    'postcss-preset-env': {
      stage: 1,
      features: {
        'nesting-rules': true,
        'custom-media-queries': true,
        'custom-properties': {
          disableDeprecationNotice: true,
        },
        'nesting-rules': true  // Enable nesting
      },
      preserve: true,
      importFrom: [
        join(
          __dirname,
          './node_modules/@example/package-web/lib/components/Inputs/DateField/style.module.css'
        ),
      ],
    },
    'postcss-hexrgba': {},
  },
};
```

This is the latest one I have tried without success.
